src/pubkey.h

```diff
@@ -300,6 +300,8 @@ class XOnlyPubKey
     SERIALIZE_METHODS(XOnlyPubKey, obj) { READWRITE(obj.m_keydata); }
 };
 
+extern const XOnlyPubKey NUMS_H;
```

Comment:
nit: Would it be better to make this a `static const` member of the `XOnlyPubKey` class?

Reply:
Can you elaborate? Not clear to me what the benefit would be.

Reply:
Mostly that `NUMS_H` is an XOnlyPubkey (rather than a CPubKey or the hex/bytes encoding of the point or something else), so maybe it would make sense to keep it tied to that class. Bit of a knee-jerk "does this really need to be in the global namespace" reaction.

Reply:
You'd write `builder.Finalize(XOnlyPubKey::NUMS_H);` which seems fine?

Reply:
Oh! I see , I misinterpreted what you were suggesting. This does seem better, will update.